pkg/oauthserver/setup_test.go

```diff
@@ -0,0 +1,141 @@
+package oauthserver_test
+
+import (
+	"encoding/base64"
+	"net/http"
+	"testing"
+	"time"
+
+	"golang.org/x/crypto/bcrypt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	fakekube "k8s.io/client-go/kubernetes/fake"
+
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	osinv1 "github.com/openshift/api/osin/v1"
+	fakeoauthclient "github.com/openshift/client-go/oauth/clientset/versioned/fake"
+	typedv1 "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+	fakeuserclient "github.com/openshift/client-go/user/clientset/versioned/fake"
+	userinformer "github.com/openshift/client-go/user/informers/externalversions"
+	bootstrap "github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
+
+	"github.com/openshift/oauth-server/pkg/config"
+	"github.com/openshift/oauth-server/pkg/oauthserver"
+	"github.com/openshift/oauth-server/pkg/userregistry/identitymapper"
+)
+
+const (
+	testPassword = "testpasswordxtestpasswordx" // must have at least len 23
+
+	clientIDKey    = "client_id"
```

Comment:
single use, inline

Reply:
I prefer to have strings or magic numbers that are constants to declare as constants, independently of the amount of usage.

But I will change the code accordingly.